transport-classes-io_uring/src/main/java/io/netty/channel/uring/AbstractIoUringStreamChannel.java

```diff
@@ -281,10 +314,17 @@ protected int scheduleWriteSingle(Object msg) {
                 ops = fileRegion.splice(fd);
             } else {
                 ByteBuf buf = (ByteBuf) msg;
-                ops = IoUringIoOps.newWrite(fd, (byte) 0, 0,
-                        IoUring.memoryAddress(buf) + buf.readerIndex(), buf.readableBytes(), nextOpsId());
+                long address = IoUring.memoryAddress(buf) + buf.readerIndex();
+                int length = buf.readableBytes();
+                short opsid = nextOpsId();
+                if (IoUring.isSendZcSupported() &&
+                        ((IoUringStreamChannelConfig) config()).shouldWriteZeroCopy(length)) {
+                    ops = IoUringIoOps.newSendZc(fd, address, length, 0, opsid, 0);
```

Comment:
My previous patch introduced a int zcFlags = Native.IORING_SEND_ZC_REPORT_USAGE to allow users to determine whether sendZC is actually performing zero-copy.
Do we still need this functionality?

Reply:
I am not sure and I think we can add it later.